src/api/network.js

```diff
@@ -2,7 +2,8 @@ const { getInstance } = require('../qclient');
 
 const Network = {
   async getPeerNodeCount() {
-    return getInstance().getPeerInfo().length || 0;
+    const nodeCount = await getInstance().getPeerInfo();
+    return await nodeCount.length || 0;
```

Comment:
see comment above. you don't need to await `nodeCount.length`. that is not an async function.

Reply:
a mistake lol thanks